UnityProject/Assets/Scripts/Shuttles/Destination.cs

```diff
@@ -16,7 +17,7 @@ public struct Destination
 	public static Destination Invalid = new Destination
 	{
 		Position = TransformState.HiddenPos,
-		Orientation = Orientation.Up
+		Orientation = new Orientation()
```

Comment:
unintended change?

Reply:
idk I just alt + enter to fix issues this whole time. will see if it's intended or not by rider.

Reply:
it probably did it so it could compile, but after you gave a namespace to our Orientation enum, this change shouldn't be needed anymore and might bring confusion in the future. Revert it back to ``Orientation.Up``, if that doesn't compile make sure you are importing our Orientation enum from ``Core``.

Reply:
Done 👍